pulpcore/app/models/exporter.py

```diff
@@ -82,10 +86,14 @@ def _export_to_file_system(self, content_artifacts):
             content_artifacts (django.db.models.QuerySet): Set of ContentArtifacts to export
 
         Raises:
-            ValidationError: When path is not in the ALLOWED_EXPORT_PATHS setting
+            RuntimeError: If there are remote artifacts or when trying to link non-fs files
```

Comment:
```suggestion
            RuntimeError: If Artifacts are not downloaded or when trying to link non-fs files
```

This would be clearer to me to avoid confusion with `RemoteArtifacts`. This suggestion is totally optional.